src/routes/postSkipSegments.js

```diff
@@ -105,7 +105,7 @@ async function autoModerateSubmission(submission, callback) {
         }
          
     } else {
-        (config.mode === 'development') && console.log("Skipped YouTube API");
+        if (config.mode === 'development') console.log("Skipped YouTube API");
```

Comment:
I think it'd be worth us implementing a logging system with levels defined (maybe linked to the mode config option).

Could work something like this: https://github.com/Joe-Dowd/users/blob/master/src/utils/logger.js
Usage:
`logger.debug("i woke up at 8:55");`
`logger.info("i woke up");`
`logger.warn("i didn't wake up");`
`logger.error("i'm dead");`

Reply:
Yep, sounds like a good idea

Reply:
In that case I say leave this to match your changes and implement the improved logging system separately.